libs/mocks/src/pools.rs

```diff
@@ -64,22 +64,6 @@ pub mod pallet {
 		) {
 			register_call!(move |(a, b, c)| f(a, b, c));
 		}
-
-		pub fn mock_bench_create_pool(f: impl Fn(T::PoolId, &T::AccountId) + 'static) {
-			register_call!(move |(a, b)| f(a, b));
-		}
-
-		pub fn mock_bench_investor_setup(
-			f: impl Fn(T::PoolId, T::AccountId, T::Balance) + 'static,
-		) {
-			register_call!(move |(a, b, c)| f(a, b, c));
-		}
-
-		pub fn mock_bench_default_investment_id(
-			f: impl Fn(T::PoolId) -> T::TrancheCurrency + 'static + 'static,
-		) {
-			register_call!(f);
-		}
```

Comment:
No longer needed if default mock implementation is default (which makes sense because you do not want to benchmark in testing)